app/components/Form/ProjectAnnualReportForm.tsx

```diff
@@ -56,10 +56,6 @@ const ProjectAnnualReportForm: React.FC<Props> = (props) => {
               newFormData
               operation
               changeStatus
-              formChangeByPreviousFormChangeId {
```

Comment:
Why did this get removed? Was it not used? It will be used after Gurjeet's PR makes it in since it's used in diffing.